src/utilities/common.tsx

```diff
@@ -130,6 +132,28 @@ export function fieldSetsToColumns (fieldSets: IFieldSetPartial[], tableModel: I
     }));
 }
 
+function nullifyValue (fieldConfig: IFieldConfig, data: IModel) {
+  const { field, nullify } = fieldConfig
+    , formValue = get(data, field)
+    , shouldNullify = nullify && !formValue && formValue !== false
+    , nullifiedValue = shouldNullify ? null : formValue
+    , isAddingNew = isObject(formValue) && !has(formValue, ID_ATTR)
+  ;
+
+  // When using the add new feature of objectSearchCreate, we should
+  // make sure to nullify the appropriate fields in the new model
+  if (isTypeObjectSearchCreate(fieldConfig) && isAddingNew) {
+    return modelFromFieldConfigs(getFieldSetFields(fieldConfig.createFields).map(fillInFieldConfig), formValue);
+  }
+
+  // When saving an address, nullify if no attributes
+  if (isTypeAddress(fieldConfig) && !some(formValue)) {
```

Comment:
Just a note that empty object is truthy. You _may_ want to consider _.isEmpty if that applies to you.